src/shared/lib/helmet/index.tsx

```diff
@@ -3,19 +3,64 @@ import Head from 'next/head'
 import { useRouter } from 'next/router'
 import { HelmetProps } from './types'
 
-export const Helmet = ({ title, description, image }: HelmetProps) => {
+export const Helmet = ({
+  title,
+  description,
+  image,
+  keywords,
+}: HelmetProps) => {
   const { asPath: path } = useRouter()
-  const canonicalUrl = process.env.NEXT_PUBLIC_HOST + path
+  const host = process.env.NEXT_PUBLIC_HOST
```

Comment:
Несильно верю, что тут всё правильно работать будет, предлагаю 
```suggestion
   const isServerSide = typeof window === 'undefined'
   const host = (isServerSide ? serverRuntimeConfig : publicRuntimeConfig).HOST
```

И импорты добавить  
```
import getConfig from 'next/config'

const { publicRuntimeConfig, serverRuntimeConfig } = getConfig()
```